doc/conf.py

```diff
@@ -30,7 +30,7 @@
 
 # General information about the project.
 project = u'PyMongo'
-copyright = u'MongoDB, Inc. 2008-{0}. MongoDB, Mongo, and the leaf logo are registered trademarks of MongoDB, Inc'.format(datetime.date.today().year)
+copyright = u'MongoDB, Inc. MongoDB, Mongo, and the leaf logo are registered trademarks of MongoDB, Inc'
```

Comment:
I see. The date is still necessary, but it can be static. Change it to "2008-present" instead of removing it.

Reply:
ok will do that